.github/workflows/publish.yml

```diff
@@ -4,17 +4,20 @@ on:
   release:
     types: [published]
 
+env:
+  NODE_VERSION: lts/gallium
+
 jobs:
   deploy:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 14.x
+          node-version: ${{ env.NODE_VERSION }}
           registry-url: 'https://registry.npmjs.org'
           scope: '@cowprotocol'
       - run: yarn --frozen-lockfile
-      - run: bash src/workflows/publish.sh
+      - run: bash workflows/publish.sh
```

Comment:
Why move `workflows` out of `src` folder?

Reply:
Because workflows it's not a source code of SDK, is an auxiliary script

Reply:
True. It's just that we usually put all code under `src` for other projects.
I'm ok with this approach, just bringing it up as this is changing the current approach